build.sbt

```diff
@@ -17,6 +17,23 @@ lazy val types = project
     )
   )
 
+lazy val iamTypes = project
+  .in(file("modules/iam-types"))
+  .settings(
+    name := "iam-types",
+    moduleName := "iam-types",
+    libraryDependencies ++= Seq(
+      "com.typesafe.akka"  %% "akka-http" % akkaHttpVersion.value,
```

Comment:
we should probably make this depend on the `http` module and introduce just `akka-http-circe`